README.md

```diff
@@ -114,6 +114,16 @@ In the above example, when the input changes, only the changeset's internal valu
 
 On rollback, all changes are dropped and the underlying Object is left untouched.
 
+## Disabling Automatic Validation
+
+The default behavior of `Changeset` is to automatically validate a field when it is set. Automatic validation can be disabled by passing `skipValidate` as on option only when creating a changeset programmatically.
```

Comment:
This actually reminds me, we should let the option be passed into the helper as well:

```js
export function changeset([obj, validations], { skipValidate }) {
  //
}
```